_config.yml

```diff
@@ -1,91 +1,114 @@
+      path: "**/*.md"
     values:
       css:
-      - /css/split-h1.css
-      - https://use.fontawesome.com/releases/v5.15.4/css/all.css
-      - /css/blog.css
-      - /css/custom.css
+        - https://use.fontawesome.com/releases/v5.15.4/css/all.css
+        - /css/custom.css
       layout: default
+      noReportProblem: true
       pageclass: cnt-wdth-lmtd
+      script: /js/gc-pg-hlpfl.js
   - scope:
-      path: "sondage"
+      path: sondage
     values:
       breadcrumbs:
-      - title: À propos de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos.html
-      - title: Analyses, sondage et commentaires
-        link: https://www.canada.ca/fr/analytique.html
-      - title: Sondage sur la réussite des tâches
-        link: sondage/index.html
+        - title: À propos de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos.html
+        - title: Analyses, sondage et commentaires
+          link: https://www.canada.ca/fr/analytique.html
+        - title: Sondage sur la réussite des tâches
+          link: sondage/index.html
   - scope:
-      path: "architecture"
+      path: architecture
     values:
       breadcrumbs:
-      - title: À propos de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos.html
-      - title: Système de conception de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
-      - title: Spécifications de l'architecture de l'information
-        link: https://conception.canada.ca/architecture/specifications-contenu-architecture-information-canada.html
+        - title: À propos de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos.html
+        - title: Système de conception de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+        - title: Spécifications de l'architecture de l'information
+          link: https://conception.canada.ca/architecture/specifications-contenu-architecture-information-canada.html
   - scope:
-      path: "configurations-conception-communes/*.md"
+      path: configurations-conception-communes/*.md
     values:
       breadcrumbs:
-      - title: À propos de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos.html
-      - title: Système de conception de Canada.ca
-        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
-      - title: Bibliothèque de modèles et de configurations de conception
-        link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception/bibliotheque-modeles.html
+        - title: À propos de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos.html
+        - title: Système de conception de Canada.ca
+          link: https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html
+        - title: Bibliothèque de modèles et de configurations de conception
```

Comment:
"Bibliothèque de modèles et de configurations de conception"  should be shortened to "Modèles et conception":

See: https://test.canada.ca/experimental/prycrane/breadcrumbs/breadcrumbs-fr.html